lib/mochaReporter.js

```diff
@@ -286,6 +280,23 @@ class ReportportalAgent extends Mocha.reporters.Base {
     }
   }
 
+  getHookStartTime(hook, hookRPType, parent) {
+    const currentParentStartTime =
+      (this.suitesInfo.get(parent) && this.suitesInfo.get(parent).startTime) || 0;
```

Comment:
May be do it once before the start time calculation?
`const suiteInfo = this.suitesInfo.get(parent);`
It seems you did what I meant [here](https://github.com/reportportal/agent-js-mocha/pull/36/files#diff-0b3fa13a94724dee7f5078043d182270R293)

Reply:
No, parent could differ from `hook.parent`. As you can see [here](https://github.com/reportportal/agent-js-mocha/pull/36/files#diff-0b3fa13a94724dee7f5078043d182270R306) parent could be equal to `hook.parent.parent` for before all hook. This time is needed to avoid shifting item to earlier time then its parent.